docs/advanced/api/authentication.rst

```diff
@@ -0,0 +1,63 @@
+==============
+Authentication
+==============
+
+The Flocker REST API uses TLS certification to secure and authenticate requests.
+This ensures an API request is both encrypted and verified to have come from an authorised user, while the corresponding response is verified to have come from the genuine cluster control service.
+
+Unlike regular HTTPS, Flocker uses TLS not just for establishing a secure connection between an API user and the cluster's control service, but also for identity verification; that is, client certification is used in place of a username and password to identify an API user to the cluster.
+
+Therefore to grant a user access to your cluster's REST API, you will need to use the ``flocker-ca`` tool installed as part of the ``flocker-cli`` package to generate a certificate and private key that is then given to the API end user.
+When you installed the ``flocker-node`` package on each of your cluster nodes, you should have already used the ``flocker-ca`` tool to generate a root certificate authority identifying your cluster.
```

Comment:
"To give a user access to a cluster's REST API, use the flocker-ca tool to generate a certificate and private key for the user. The flocker-ca tool is installed as part of the flocker-cli package."